app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from typing import Union
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
```

Comment:
Use ( ) while importing multiple classes/modules, follow example formatting imports in checklist